backend.py

```diff
@@ -493,9 +495,14 @@ def setTimeManually(localtime, set_time_dialog_dismissed, timezone):
     util.setLocalTime(timestr, timezone=timezone)
     assert util.runCmd2(['hwclock', '--utc', '--systohc']) == 0
 
-def configureNTP(mounts, ntp_servers):
+def setDHCPNTP(mounts, ntp_config_method):
+    script = os.path.join(mounts['root'], "etc/dhcp/dhclient.d/chrony.sh")
+    permission = "a+x" if ntp_config_method == "dhcp" else "a-x"
+    assert util.runCmd2(["chmod", permission, script]) == 0
```

Comment:
It would be better to use `os.chmod` instead of shelling out to do this.

Otherwise, this looks good to me.